.gitignore

```diff
@@ -46,4 +47,6 @@ gradle-app.setting
 .classpath
 
 .idea/
-out/
\ No newline at end of file
+out/
+bin/
+.vscode/
```

Comment:
I've added some excludes for files that we're previously added and removed in this PR.